vtex/faststore
# chore(store-ui): Slider atom - change input component

## What's the purpose of this pull request?
Use store-ui Input component instead of HTML input tag to keep the design system pattern.

## How it works? 
- Use `<Input />` instead of HTML input tag in [Slider](https://github.com/vtex/faststore/blob/master/packages/store-ui/src/atoms/Slider/Slider.tsx) atom.
- Update 'CSS Selector' section Slider of the storybook page documentation.

## How to test it?
The Slider behavior has to be the same as the last implementation.
- Only check if it's possible to navigate to Input CSS selectors linked on Slider's storybook page.

## References
_
